### PR TITLE
Added image for locations not found

### PR DESCRIPTION
**Pull request by siddnikh**
Solves issue #3045

### CODE REVIEW

**Review by naman114 — CHANGES_REQUESTED** [review]
Hey @siddnikh 

Could the icon be vertically centered? It leaves a lot of blank space on smaller screens

image

**Comment by nihal467** [comment]
@siddnikh 
image

- [ ] the whole page in normal view itself is stretched and we need to scroll the page down, adjust the page , in a way that we don't need to scroll the page
- [ ] the icon doesn't feel good, take an opinion from other in EOD call or slack channel before working on the icon again

**Comment by nihal467** [comment]
@siddnikh 
image

the page is having scroller in normal view, readjust the icon size and position so that, the scroller doesn't appear on the page

**Comment by nihal467** [comment]
@siddnikh 
image

the page is still having the scroller, reduce the size of the icon and bring it little bit upward

**Comment by siddnikh** [comment]
> @siddnikh image
> 
> the page is still having the scroller, reduce the size of the icon and bring it little bit upward

I have tested on all device dimensions on my laptop. I hope this works now. If it does not, can you tell me your screen dimensions? Thanks. @nihal467

File: src/Components/Facility/LocationManagement.tsx [hunk 1]
```diff
@@ -122,27 +122,36 @@ export const LocationManagement = (props: LocationManagementProps) => {
...
   } else if (locations && locations.length === 0) {
     locationsList = (
-      <p className="bg-white flex justify-center text-2xl w-full font-bold px-5 py-5 border-b border-gray-200 text-center text-gray-500">
-        No locations available
-      </p>
+      <div className="h-full content-center grid justify-items-center mt-40">
+        <div className="flex justify-center w-full items-center">
+          <img
+            className="w-3/12 sm:w-3/12 md:w-3/12 lg:w-2/12 xl:w-2/12 2xl:w-2/12"
+            src={"https://cdn.coronasafe.network/location.png"}
```
Comment (khavinshankar) [inline src/Components/Facility/LocationManagement.tsx:140]: @siddnikh can you use an icon instead of an image, also reduce the size of the icon a little

**Comment by nihal467** [comment]
@siddnikh what is the update on this PR

**Comment by siddnikh** [comment]
@nihal467 https://deploy-preview-3275--care-net.netlify.app/facility/6fcc52a3-f917-4e5c-ab33-d9af2cd26a16/location
here, please try this out and lmk if there still exists any issue on your side.

File: src/Components/Facility/LocationManagement.tsx (hunk 1 above)
Comment (siddnikh) [reply src/Components/Facility/LocationManagement.tsx:140]: sure, let me try making the image smaller.

**Comment by nihal467** [comment]
@siddnikh 
image

@siddnikh marking it as team discussion required for multiple reason

- we are moving towards careicons, is it okay to use this icon
- does the UI looks good for approval

**Comment by nihal467** [comment]
@siddnikh 
image


- [ ] the icon size looks really big, make it smaller
- [ ] the icon seem to be cropped in the bottom, doesn't look good

**Comment by siddnikh** [comment]
@nihal467 I will make a new PR for this, it's an old branch so I'm having trouble merging it with develop.

**Pull request closed without merging**